.github/workflows/windows-release.yml

```
@@ -1,11 +1,16 @@
   push:
     tags:
       - 'v*.*.*'
+  workflow_dispatch:
+
+# Minimal permissions — principle of least privilege.
+permissions:
+  contents: write
+  actions: read
 
-jobs:
   build-and-release:
```

actionlint (1.7.12)

[error] unknown permission scope "build-and-release". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)